content/network/drivers/host.md

```diff
@@ -42,10 +42,63 @@ daemon's host network and ports. This creates some extra limitations. For instan
 if a service container binds to port 80, only one service container can run on a
 given swarm node.
 
+## Docker Desktop
+
+Since Docker Desktop 4.29 host networking is also supported on Docker Desktop for Mac,
+Windows, and Linux as a beta feature for users with a paid subscription
+(Docker Pro or higher).
+
+This feature works in both directions. That means that you can
+access a server that is running in a container from your host and you can access
+servers running on your host from any container that is started with host
+networking enabled. TCP as well as UDP are supported as communication protocols.
+
+### Examples
+
+This command starts netcat in a container to listen on port 8000:
+
+```
+docker run --rm -it --net=host nicolaka/netshoot nc -lkv 0.0.0.0 8000
+```
+
+Port 8000 will then be available on the host and you can connect to it with this
+command from another terminal:
+
+```
+nc localhost 8000
+```
+
+What you type in here will then appear on the terminal where the container is
+running.
+
+To access a service running on the container you can start a container with
```

Comment:
> To access a service running on the container

Did you mean "To access a service running on the host from the container, you can start ..."